tfhe/src/boolean/engine/bootstrapping.rs

```diff
@@ -439,8 +439,8 @@ impl From<CompressedServerKey> for ServerKey {
             pbs_order,
         } = compressed_server_key;
 
-        let key_switching_key = key_switching_key.decompress_into_lwe_keyswitch_key();
-        let standard_bootstrapping_key = bootstrapping_key.decompress_into_lwe_bootstrap_key();
+        let key_switching_key = key_switching_key.par_decompress_into_lwe_keyswitch_key();
+        let standard_bootstrapping_key = bootstrapping_key.par_decompress_into_lwe_bootstrap_key();
```

Comment:
rayon::join ? 😈

Reply:
unlikely to make a big difference, but I can add it yes